source/fundamentals/linq.txt

```diff
@@ -497,6 +497,82 @@ The following is a subset of the returned results:
        ]
     }
 
+$vectorSearch
+~~~~~~~~~~~~~
+
+The ``$vectorSearch`` aggregation stage performs an Approximate Nearest Neighbor search
+on an Atlas Vector Search index in the specified field.
```

Comment:
S: I think it's more common to search "on" a field than "in", unless you're searching within a single value of a single field for a substring or something

```suggestion
on an Atlas Vector Search index on the specified field.
```